common/scala/src/main/scala/whisk/core/containerpool/logging/SplunkLogStore.scala

```diff
@@ -100,8 +100,8 @@ class SplunkLogStore(
         "exec_mode" -> "oneshot",
         "search" -> search,
         "output_mode" -> "json",
-        "earliest_time" -> formatter.format(activation.start),
-        "latest_time" -> formatter.format(activation.end))).toEntity
```

Comment:
I originally had this in there to limit the time range as much as possible for the splunk query, since the activation may have run days ago, lasted 10ms, and there may be millions of log events since then. 
I'm not positive that splunk does better with a narrower time range, but I assumed it does - WDYT? 
I could add it back in, but with end time + 5s to avoid any issues with collecting logs on short activations?